docs/user/metacat/source/intro.rst

```diff
@@ -98,14 +89,13 @@ that have been created by others. Specifically,
 * Metacat's Web interface facilitates the input and retrieval of data 
 * Metacat's optional mapping functionality enables you to query and visualize the geographic coverage of stored data sets
 * Metacat's replication feature ensures that all Metacat data and metadata is stored safely on multiple Metacat servers
-* The Metacat interface can be easily extended and customized via Web forms, themes, and/or user-developed client tools in Java and other languages
-* The Metacat harvester automates the process of retrieving and storing EML documents from one or more sites
+* Metacat's default web interface can be easily modified through metacatui, which is shipped with Metacat.
```

Comment:
suggest:
~~Metacat's default web interface can be easily modified through metacatui, which is shipped with Metacat.~~
Metacat's default web interface ("metacatui") can be easily modified.
(and add a link to the relevant metacatui docs?)